src/app.jsx

```diff
@@ -120,12 +124,14 @@ class App extends React.Component {
     };
 
     // /* We retrieve the position of the cursor from the URL if exists */
-    // if(this.router.params.toJSON().timelineDate) {
-    //   const date = moment.utc(this.router.params.toJSON().timelineDate, 'YYYY-MM-DD');
-    //   if(date.isValid()) {
-    //     timelineParams.cursorPosition = date.toDate();
-    //   }
-    // }
+    if(router.params.toJSON().timelineDate) {
+      const date = moment.utc(router.params.toJSON().timelineDate, 'YYYY-MM-DD');
+      if(date.isValid()) {
+        timelineParams.cursorPosition = date.toDate();
+      }
+    } else {
+      timelineParams.cursorPosition = this.state.timelineDate;
+    }
 
```

Comment:
We already have a function where we retrieve all router params and set them as app state. You'd only need to take that param from app state.